great_expectations/jupyter_ux/__init__.py

```diff
@@ -189,7 +190,11 @@ def show_available_data_asset_names(context, data_source_name=None):
     """
 
     print("Inspecting your data sources. This may take a moment...")
-    expectation_suite_keys = context.list_expectation_suites()
+    expectation_suite_keys = []
+    try:
+        expectation_suite_keys = context.list_expectation_suites()
+    except ge_exceptions.InvalidConfigError:
+        pass
```

Comment:
You could pull out line 193 and put it in place of 197
```suggestion
        expectation_suite_keys = []
```

Reply:
Since this code existed before, I would prefer that it is addressed in a different PR.